pkg/virt-controller/watch/migration.go

```diff
@@ -540,13 +554,13 @@ func (c *MigrationController) updateStatus(migration *virtv1.VirtualMachineInsta
 		}
 	}
 
-	if !equality.Semantic.DeepEqual(migration.Status, migrationCopy.Status) {
-		err := c.statusUpdater.UpdateStatus(migrationCopy)
+	if !equality.Semantic.DeepEqual(migration.Finalizers, migrationCopy.Finalizers) {
```

Comment:
Why does this need to have priority?

Reply:
I honestly don't remember why I did that :open_mouth: 
In any case, I've reverted this and all relevant functional tests pass.

Thanks for catching that!